components/campaigns/AnnoucementHero.js

```diff
@@ -1,26 +1,25 @@
-// import { useRouter } from 'next/router'
-// import YouTube from 'react-youtube-embed'
-// import Button from '../buttons/Button'
+import { useRouter } from 'next/router'
+import Button from '../buttons/Button'
+import Heading from '../typography/Heading'
+import Paragraph from '../typography/Paragraph'
 
 export default function AnnouncementHero({ className = '', small = false}) {
-  return null;
 
-  // const { pathname } = useRouter()
+  const { pathname } = useRouter()
 
-  // if (pathname === '/blog/events2021') return null
+  if (pathname === '/blog/events2021') return null
 
-  // return (
-  //   <div className={`bg-gray-50 border border-gray-200 py-6  ${className} ${small ? 'mb-0' : 'mb-12'}`}>
-  //     <h2 className="text-xl md:text-3xl font-bold countdown-text-gradient">AsyncAPI Conference Day 3 is running!</h2>
-  //     <div className='px-4 mt-4 mx-auto max-w-7xl'>
-  //       <YouTube
-  //         id="3EeMHhbwyOQ"
-  //       />
-  //     </div>
-  //     <div className="mt-8 pb-2 space-x-2">
-  //       <Button href="https://www.asyncapi.com/slack-invite" target="_blank" text="Comment and ask questions" />
-  //       <Button bgClassName="bg-none border border-gray-200 text-gray-800 hover:text-gray-700 shadow-none" href="https://conference.asyncapi.com/#schedule" target="_blank" text="Check schedule" />
-  //     </div>
-  //   </div>
-  // )
+  return (
+    <div className={`bg-gray-50 border border-gray-200 py-6 rounded ${className} ${small ? 'mb-4' : 'mb-12'}`}>
+      <Heading className="text-xl md:text-3xl font-bold countdown-text-gradient" level="h2" typeStyle="heading-lg countdown-text-gradient">
```

Comment:
```suggestion
      <Heading className="countdown-text-gradient" level="h2" typeStyle="heading-lg">
```

No need for those extra type class names, the `heading-lg` typeStyle prop handles all the styles automatically 😄  & we should remove `countdown-text-gradient` from the `typeStyle` prop as it was breaking the prop.